{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  adding: elasticsearch-cognito.py (deflated 66%)\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\n",
      "  adding: check_transcribe.py (deflated 56%)\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\n",
      "  adding: elasticsearch-cognito.py (deflated 69%)\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\n",
      "  adding: elasticsearch-6.2.0.dist-info/ (stored 0%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/WHEEL (deflated 14%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/metadata.json (deflated 61%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/top_level.txt (stored 0%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/RECORD (deflated 57%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/DESCRIPTION.rst (deflated 58%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/METADATA (deflated 62%)\n",
      "  adding: elasticsearch-6.2.0.dist-info/INSTALLER (stored 0%)\n",
      "  adding: certifi/ (stored 0%)\n",
      "  adding: certifi/__main__.py (deflated 5%)\n",
      "  adding: certifi/__init__.py (deflated 3%)\n",
      "  adding: certifi/cacert.pem (deflated 46%)\n",
      "  adding: certifi/core.py (deflated 49%)\n",
      "  adding: elasticsearch/ (stored 0%)\n",
      "  adding: elasticsearch/compat.py (deflated 54%)\n",
      "  adding: elasticsearch/client/ (stored 0%)\n",
      "  adding: elasticsearch/client/remote.py (deflated 37%)\n",
      "  adding: elasticsearch/client/nodes.py (deflated 75%)\n",
      "  adding: elasticsearch/client/utils.py (deflated 61%)\n",
      "  adding: elasticsearch/client/cluster.py (deflated 76%)\n",
      "  adding: elasticsearch/client/xpack/ (stored 0%)\n",
      "  adding: elasticsearch/client/xpack/migration.py (deflated 60%)\n",
      "  adding: elasticsearch/client/xpack/monitoring.py (deflated 49%)\n",
      "  adding: elasticsearch/client/xpack/security.py (deflated 89%)\n",
      "  adding: elasticsearch/client/xpack/ml.py (deflated 87%)\n",
      "  adding: elasticsearch/client/xpack/__init__.py (deflated 65%)\n",
      "  adding: elasticsearch/client/xpack/license.py (deflated 66%)\n",
      "  adding: elasticsearch/client/xpack/graph.py (deflated 52%)\n",
      "  adding: elasticsearch/client/xpack/deprecation.py (deflated 44%)\n",
      "  adding: elasticsearch/client/xpack/watcher.py (deflated 84%)\n",
      "  adding: elasticsearch/client/__init__.py (deflated 86%)\n",
      "  adding: elasticsearch/client/indices.py (deflated 87%)\n",
      "  adding: elasticsearch/client/tasks.py (deflated 73%)\n",
      "  adding: elasticsearch/client/ingest.py (deflated 76%)\n",
      "  adding: elasticsearch/client/cat.py (deflated 90%)\n",
      "  adding: elasticsearch/client/snapshot.py (deflated 85%)\n",
      "  adding: elasticsearch/helpers/ (stored 0%)\n",
      "  adding: elasticsearch/helpers/__init__.py (deflated 71%)\n",
      "  adding: elasticsearch/helpers/test.py (deflated 60%)\n",
      "  adding: elasticsearch/__init__.py (deflated 48%)\n",
      "  adding: elasticsearch/transport.py (deflated 70%)\n",
      "  adding: elasticsearch/connection/ (stored 0%)\n",
      "  adding: elasticsearch/connection/base.py (deflated 66%)\n",
      "  adding: elasticsearch/connection/http_requests.py (deflated 66%)\n",
      "  adding: elasticsearch/connection/__init__.py (deflated 38%)\n",
      "  adding: elasticsearch/connection/pooling.py (deflated 59%)\n",
      "  adding: elasticsearch/connection/http_urllib3.py (deflated 67%)\n",
      "  adding: elasticsearch/exceptions.py (deflated 68%)\n",
      "  adding: elasticsearch/serializer.py (deflated 70%)\n",
      "  adding: elasticsearch/connection_pool.py (deflated 70%)\n",
      "  adding: chardet/ (stored 0%)\n",
      "  adding: chardet/compat.py (deflated 48%)\n",
      "  adding: chardet/eucjpprober.py (deflated 65%)\n",
      "  adding: chardet/version.py (deflated 24%)\n",
      "  adding: chardet/mbcharsetprober.py (deflated 64%)\n",
      "  adding: chardet/charsetgroupprober.py (deflated 70%)\n",
      "  adding: chardet/euckrprober.py (deflated 53%)\n",
      "  adding: chardet/sbcharsetprober.py (deflated 66%)\n",
      "  adding: chardet/langgreekmodel.py (deflated 81%)\n",
      "  adding: chardet/mbcssm.py (deflated 87%)\n",
      "  adding: chardet/jisfreq.py (deflated 52%)\n",
      "  adding: chardet/escprober.py (deflated 66%)\n",
      "  adding: chardet/big5freq.py (deflated 53%)\n",
      "  adding: chardet/langcyrillicmodel.py (deflated 83%)\n",
      "  adding: chardet/charsetprober.py (deflated 61%)\n",
      "  adding: chardet/mbcsgroupprober.py (deflated 56%)\n",
      "  adding: chardet/langthaimodel.py (deflated 75%)\n",
      "  adding: chardet/big5prober.py (deflated 53%)\n",
      "  adding: chardet/gb2312prober.py (deflated 53%)\n",
      "  adding: chardet/langbulgarianmodel.py (deflated 78%)\n",
      "  adding: chardet/__init__.py (deflated 53%)\n",
      "  adding: chardet/jpcntx.py (deflated 75%)\n",
      "  adding: chardet/escsm.py (deflated 85%)\n",
      "  adding: chardet/utf8prober.py (deflated 60%)\n",
      "  adding: chardet/codingstatemachine.py (deflated 60%)\n",
      "  adding: chardet/langhungarianmodel.py (deflated 78%)\n",
      "  adding: chardet/gb2312freq.py (deflated 51%)\n",
      "  adding: chardet/euctwprober.py (deflated 53%)\n",
      "  adding: chardet/chardistribution.py (deflated 75%)\n",
      "  adding: chardet/euckrfreq.py (deflated 54%)\n",
      "  adding: chardet/hebrewprober.py (deflated 66%)\n",
      "  adding: chardet/sbcsgroupprober.py (deflated 67%)\n",
      "  adding: chardet/latin1prober.py (deflated 69%)\n",
      "  adding: chardet/langturkishmodel.py (deflated 75%)\n",
      "  adding: chardet/langhebrewmodel.py (deflated 78%)\n",
      "  adding: chardet/sjisprober.py (deflated 65%)\n",
      "  adding: chardet/euctwfreq.py (deflated 53%)\n",
      "  adding: chardet/universaldetector.py (deflated 72%)\n",
      "  adding: chardet/cp949prober.py (deflated 53%)\n",
      "  adding: chardet/cli/ (stored 0%)\n",
      "  adding: chardet/cli/chardetect.py (deflated 57%)\n",
      "  adding: chardet/cli/__init__.py (stored 0%)\n",
      "  adding: chardet/enums.py (deflated 58%)\n",
      "  adding: idna-2.6.dist-info/ (stored 0%)\n",
      "  adding: idna-2.6.dist-info/WHEEL (deflated 14%)\n",
      "  adding: idna-2.6.dist-info/metadata.json (deflated 54%)\n",
      "  adding: idna-2.6.dist-info/top_level.txt (stored 0%)\n",
      "  adding: idna-2.6.dist-info/RECORD (deflated 42%)\n",
      "  adding: idna-2.6.dist-info/DESCRIPTION.rst (deflated 61%)\n",
      "  adding: idna-2.6.dist-info/METADATA (deflated 62%)\n",
      "  adding: idna-2.6.dist-info/INSTALLER (stored 0%)\n",
      "  adding: upload_to_elasticsearch.py (deflated 59%)\n",
      "  adding: idna/ (stored 0%)\n",
      "  adding: idna/compat.py (deflated 38%)\n",
      "  adding: idna/codec.py (deflated 76%)\n",
      "  adding: idna/__init__.py (deflated 12%)\n",
      "  adding: idna/core.py (deflated 76%)\n",
      "  adding: idna/uts46data.py (deflated 81%)\n",
      "  adding: idna/package_data.py (stored 0%)\n",
      "  adding: idna/intranges.py (deflated 58%)\n",
      "  adding: idna/idnadata.py (deflated 78%)\n",
      "  adding: requests-2.18.4.dist-info/ (stored 0%)\n",
      "  adding: requests-2.18.4.dist-info/WHEEL (deflated 14%)\n",
      "  adding: requests-2.18.4.dist-info/metadata.json (deflated 57%)\n",
      "  adding: requests-2.18.4.dist-info/top_level.txt (stored 0%)\n",
      "  adding: requests-2.18.4.dist-info/RECORD (deflated 48%)\n",
      "  adding: requests-2.18.4.dist-info/DESCRIPTION.rst (deflated 66%)\n",
      "  adding: requests-2.18.4.dist-info/METADATA (deflated 66%)\n",
      "  adding: requests-2.18.4.dist-info/INSTALLER (stored 0%)\n",
      "  adding: urllib3-1.22.dist-info/ (stored 0%)\n",
      "  adding: urllib3-1.22.dist-info/WHEEL (deflated 14%)\n",
      "  adding: urllib3-1.22.dist-info/metadata.json (deflated 58%)\n",
      "  adding: urllib3-1.22.dist-info/top_level.txt (stored 0%)\n",
      "  adding: urllib3-1.22.dist-info/RECORD (deflated 56%)\n",
      "  adding: urllib3-1.22.dist-info/DESCRIPTION.rst (deflated 63%)\n",
      "  adding: urllib3-1.22.dist-info/METADATA (deflated 64%)\n",
      "  adding: urllib3-1.22.dist-info/INSTALLER (stored 0%)\n",
      "  adding: aws_requests_auth-0.4.1.dist-info/ (stored 0%)\n",
      "  adding: aws_requests_auth-0.4.1.dist-info/WHEEL (deflated 1%)\n",
      "  adding: aws_requests_auth-0.4.1.dist-info/top_level.txt (stored 0%)\n",
      "  adding: aws_requests_auth-0.4.1.dist-info/RECORD (deflated 46%)\n",
      "  adding: aws_requests_auth-0.4.1.dist-info/METADATA (deflated 46%)\n",
      "  adding: aws_requests_auth-0.4.1.dist-info/INSTALLER (stored 0%)\n",
      "  adding: requests/ (stored 0%)\n",
      "  adding: requests/compat.py (deflated 64%)\n",
      "  adding: requests/certs.py (deflated 35%)\n",
      "  adding: requests/cookies.py (deflated 73%)\n",
      "  adding: requests/help.py (deflated 69%)\n",
      "  adding: requests/structures.py (deflated 62%)\n",
      "  adding: requests/sessions.py (deflated 72%)\n",
      "  adding: requests/utils.py (deflated 68%)\n",
      "  adding: requests/__init__.py (deflated 59%)\n",
      "  adding: requests/models.py (deflated 71%)\n",
      "  adding: requests/exceptions.py (deflated 66%)\n",
      "  adding: requests/__version__.py (deflated 41%)\n",
      "  adding: requests/_internal_utils.py (deflated 51%)\n",
      "  adding: requests/status_codes.py (deflated 62%)\n",
      "  adding: requests/packages.py (deflated 42%)\n",
      "  adding: requests/api.py (deflated 75%)\n",
      "  adding: requests/hooks.py (deflated 51%)\n",
      "  adding: requests/adapters.py (deflated 75%)\n",
      "  adding: requests/auth.py (deflated 70%)\n",
      "  adding: aws_requests_auth/ (stored 0%)\n",
      "  adding: aws_requests_auth/aws_auth.py (deflated 69%)\n",
      "  adding: aws_requests_auth/__init__.py (stored 0%)\n",
      "  adding: aws_requests_auth/boto_utils.py (deflated 61%)\n",
      "  adding: urllib3/ (stored 0%)\n",
      "  adding: urllib3/filepost.py (deflated 63%)\n",
      "  adding: urllib3/util/ (stored 0%)\n",
      "  adding: urllib3/util/selectors.py (deflated 76%)\n",
      "  adding: urllib3/util/connection.py (deflated 60%)\n",
      "  adding: urllib3/util/ssl_.py (deflated 66%)\n",
      "  adding: urllib3/util/retry.py (deflated 69%)\n",
      "  adding: urllib3/util/timeout.py (deflated 70%)\n",
      "  adding: urllib3/util/__init__.py (deflated 60%)\n",
      "  adding: urllib3/util/wait.py (deflated 65%)\n",
      "  adding: urllib3/util/request.py (deflated 65%)\n",
      "  adding: urllib3/util/response.py (deflated 58%)\n",
      "  adding: urllib3/util/url.py (deflated 64%)\n",
      "  adding: urllib3/connection.py (deflated 68%)\n",
      "  adding: urllib3/poolmanager.py (deflated 71%)\n",
      "  adding: urllib3/contrib/ (stored 0%)\n",
      "  adding: urllib3/contrib/_securetransport/ (stored 0%)\n",
      "  adding: urllib3/contrib/_securetransport/__init__.py (stored 0%)\n",
      "  adding: urllib3/contrib/_securetransport/low_level.py (deflated 68%)\n",
      "  adding: urllib3/contrib/_securetransport/bindings.py (deflated 78%)\n",
      "  adding: urllib3/contrib/appengine.py (deflated 71%)\n",
      "  adding: urllib3/contrib/securetransport.py (deflated 73%)\n",
      "  adding: urllib3/contrib/socks.py (deflated 70%)\n",
      "  adding: urllib3/contrib/__init__.py (stored 0%)\n",
      "  adding: urllib3/contrib/pyopenssl.py (deflated 68%)\n",
      "  adding: urllib3/contrib/ntlmpool.py (deflated 68%)\n",
      "  adding: urllib3/__init__.py (deflated 60%)\n",
      "  adding: urllib3/connectionpool.py (deflated 72%)\n",
      "  adding: urllib3/request.py (deflated 67%)\n",
      "  adding: urllib3/response.py (deflated 72%)\n",
      "  adding: urllib3/packages/ (stored 0%)\n",
      "  adding: urllib3/packages/ordered_dict.py (deflated 69%)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  adding: urllib3/packages/six.py (deflated 75%)\r\n",
      "  adding: urllib3/packages/__init__.py (deflated 27%)\r\n",
      "  adding: urllib3/packages/ssl_match_hostname/ (stored 0%)\r\n",
      "  adding: urllib3/packages/ssl_match_hostname/_implementation.py (deflated 61%)\r\n",
      "  adding: urllib3/packages/ssl_match_hostname/__init__.py (deflated 54%)\r\n",
      "  adding: urllib3/packages/backports/ (stored 0%)\r\n",
      "  adding: urllib3/packages/backports/makefile.py (deflated 60%)\r\n",
      "  adding: urllib3/packages/backports/__init__.py (stored 0%)\r\n",
      "  adding: urllib3/exceptions.py (deflated 67%)\r\n",
      "  adding: urllib3/_collections.py (deflated 69%)\r\n",
      "  adding: urllib3/fields.py (deflated 69%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/ (stored 0%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/WHEEL (deflated 13%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/metadata.json (deflated 55%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/top_level.txt (stored 0%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/RECORD (deflated 43%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/DESCRIPTION.rst (deflated 52%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/LICENSE.txt (deflated 43%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/METADATA (deflated 58%)\r\n",
      "  adding: certifi-2018.4.16.dist-info/INSTALLER (stored 0%)\r\n",
      "  adding: chardet-3.0.4.dist-info/ (stored 0%)\r\n",
      "  adding: chardet-3.0.4.dist-info/WHEEL (deflated 14%)\r\n",
      "  adding: chardet-3.0.4.dist-info/metadata.json (deflated 57%)\r\n",
      "  adding: chardet-3.0.4.dist-info/top_level.txt (stored 0%)\r\n",
      "  adding: chardet-3.0.4.dist-info/RECORD (deflated 52%)\r\n",
      "  adding: chardet-3.0.4.dist-info/entry_points.txt (deflated 20%)\r\n",
      "  adding: chardet-3.0.4.dist-info/DESCRIPTION.rst (deflated 53%)\r\n",
      "  adding: chardet-3.0.4.dist-info/METADATA (deflated 58%)\r\n",
      "  adding: chardet-3.0.4.dist-info/INSTALLER (stored 0%)\r\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\r\n",
      "  adding: process_transcription.py (deflated 76%)\r\n",
      "  adding: common_lib.py (deflated 61%)\r\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\r\n",
      "  adding: lambda_function.py (deflated 53%)\r\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\r\n",
      "  adding: start_transcription.py (deflated 62%)\r\n",
      "  adding: common_lib.py (deflated 61%)\r\n",
      "/home/ec2-user/SageMaker/voice-of-customer-integration-handson/part2_solution\r\n"
     ]
    }
   ],
   "source": [
    "!bash build.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "bucket = 'voice-of-customer-gokazu-20201027'\n",
    "# ex) voice-of-customer-{name}-{date}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "make_bucket failed: s3://voice-of-customer-gokazu-20201027 An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it.\r\n"
     ]
    }
   ],
   "source": [
    "!aws s3 mb s3://{bucket}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: .ipynb_checkpoints/setup-checkpoint.ipynb to s3://voice-of-customer-gokazu-20201027/asset/.ipynb_checkpoints/setup-checkpoint.ipynb An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: ./build.sh to s3://voice-of-customer-gokazu-20201027/asset/build.sh An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: ci/base-pipeline.json to s3://voice-of-customer-gokazu-20201027/asset/ci/base-pipeline.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: ci/config.yml to s3://voice-of-customer-gokazu-20201027/asset/ci/config.yml An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: ./default_dashboards.json to s3://voice-of-customer-gokazu-20201027/asset/default_dashboards.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/CheckTranscriptionJob.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/CheckTranscriptionJob.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/CheckElasticsearchStatus.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/CheckElasticsearchStatus.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/ElasticsearchCognito.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/ElasticsearchCognito.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/IndexS3TranscriptionDataIntoES.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/IndexS3TranscriptionDataIntoES.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/StartTranscriptionJob.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/StartTranscriptionJob.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/CheckTranscriptionJob/check_transcribe.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/CheckTranscriptionJob/check_transcribe.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/CheckElasticsearchStatus/elasticsearch-cognito.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/CheckElasticsearchStatus/elasticsearch-cognito.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/ElasticsearchCognito/elasticsearch-cognito.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/ElasticsearchCognito/elasticsearch-cognito.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/ProcessTranscription.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/ProcessTranscription.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/DESCRIPTION.rst to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/DESCRIPTION.rst An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth/aws_auth.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth/aws_auth.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth/boto_utils.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth/boto_utils.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/metadata.json to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/metadata.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/LICENSE.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/LICENSE.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi-2018.4.16.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/packages/S3ProcessNewAudio.zip to s3://voice-of-customer-gokazu-20201027/asset/functions/packages/S3ProcessNewAudio.zip An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/aws_requests_auth-0.4.1.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi/cacert.pem to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi/cacert.pem An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi/__main__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi/__main__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/DESCRIPTION.rst to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/DESCRIPTION.rst An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/entry_points.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/entry_points.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/certifi/core.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/certifi/core.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/metadata.json to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet-3.0.4.dist-info/metadata.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/big5freq.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/big5freq.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/big5prober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/big5prober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/cli/chardetect.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/cli/chardetect.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/chardistribution.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/chardistribution.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/charsetprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/charsetprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/charsetgroupprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/charsetgroupprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/cli/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/cli/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/cp949prober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/cp949prober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/enums.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/enums.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/codingstatemachine.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/codingstatemachine.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/compat.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/compat.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/escsm.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/escsm.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/escprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/escprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/eucjpprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/eucjpprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/euckrprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/euckrprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/euctwfreq.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/euctwfreq.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/euctwprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/euctwprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/euckrfreq.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/euckrfreq.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/jisfreq.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/jisfreq.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/gb2312prober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/gb2312prober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/jpcntx.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/jpcntx.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/hebrewprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/hebrewprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/gb2312freq.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/gb2312freq.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langgreekmodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langgreekmodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langthaimodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langthaimodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langbulgarianmodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langbulgarianmodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langcyrillicmodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langcyrillicmodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langhebrewmodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langhebrewmodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langturkishmodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langturkishmodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/latin1prober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/latin1prober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/mbcsgroupprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/mbcsgroupprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/mbcharsetprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/mbcharsetprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/langhungarianmodel.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/langhungarianmodel.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/mbcssm.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/mbcssm.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/sbcharsetprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/sbcharsetprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/sbcsgroupprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/sbcsgroupprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/universaldetector.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/universaldetector.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/sjisprober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/sjisprober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/DESCRIPTION.rst to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/DESCRIPTION.rst An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/metadata.json to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/metadata.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/version.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/version.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/chardet/utf8prober.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/chardet/utf8prober.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch-6.2.0.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/nodes.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/nodes.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/cluster.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/cluster.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/cat.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/cat.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/indices.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/indices.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/remote.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/remote.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/snapshot.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/snapshot.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/tasks.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/tasks.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/deprecation.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/deprecation.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/ingest.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/ingest.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/utils.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/utils.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/license.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/license.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/graph.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/graph.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/migration.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/migration.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/compat.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/compat.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/monitoring.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/monitoring.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/watcher.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/watcher.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/http_requests.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/http_requests.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/security.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/security.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/base.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/base.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/http_urllib3.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/http_urllib3.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/pooling.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection/pooling.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection_pool.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/connection_pool.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/exceptions.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/exceptions.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/helpers/test.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/helpers/test.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/helpers/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/helpers/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/DESCRIPTION.rst to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/DESCRIPTION.rst An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/transport.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/transport.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/serializer.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/serializer.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/metadata.json to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/metadata.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna-2.6.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/codec.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/codec.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/compat.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/compat.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/core.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/core.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/intranges.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/intranges.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/idnadata.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/idnadata.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/package_data.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/package_data.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/idna/uts46data.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/idna/uts46data.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/ml.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/elasticsearch/client/xpack/ml.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/DESCRIPTION.rst to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/DESCRIPTION.rst An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/metadata.json to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/metadata.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/__version__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/__version__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/_internal_utils.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/_internal_utils.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests-2.18.4.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/certs.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/certs.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/api.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/api.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/compat.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/compat.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/auth.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/auth.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/help.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/help.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/adapters.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/adapters.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/cookies.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/cookies.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/exceptions.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/exceptions.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/packages.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/packages.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/models.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/models.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/hooks.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/hooks.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/status_codes.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/status_codes.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/structures.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/structures.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/sessions.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/sessions.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/upload_to_elasticsearch.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/upload_to_elasticsearch.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/requests/utils.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/requests/utils.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/INSTALLER to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/INSTALLER An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/WHEEL to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/WHEEL An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/RECORD to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/RECORD An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/METADATA to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/METADATA An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/connection.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/connection.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/_collections.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/_collections.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/connectionpool.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/connectionpool.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/top_level.txt to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/top_level.txt An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/DESCRIPTION.rst to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/DESCRIPTION.rst An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/metadata.json to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3-1.22.dist-info/metadata.json An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/_securetransport/bindings.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/_securetransport/bindings.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/_securetransport/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/_securetransport/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/_securetransport/low_level.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/_securetransport/low_level.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/socks.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/socks.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/ntlmpool.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/ntlmpool.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/pyopenssl.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/pyopenssl.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/appengine.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/appengine.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/exceptions.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/exceptions.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/securetransport.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/contrib/securetransport.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/fields.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/fields.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/filepost.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/filepost.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/backports/makefile.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/backports/makefile.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/six.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/six.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/backports/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/backports/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/ordered_dict.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/ordered_dict.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/ssl_match_hostname/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/ssl_match_hostname/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/request.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/request.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/ssl_match_hostname/_implementation.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/packages/ssl_match_hostname/_implementation.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/poolmanager.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/poolmanager.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/response.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/response.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/__init__.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/__init__.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/request.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/request.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/selectors.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/selectors.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/response.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/response.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/retry.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/retry.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/connection.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/connection.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/ssl_.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/ssl_.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/timeout.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/timeout.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/url.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/url.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/ProcessTranscription/common_lib.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/ProcessTranscription/common_lib.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/wait.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/IndexS3TranscriptionDataIntoES/urllib3/util/wait.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/ProcessTranscription/process_transcription.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/ProcessTranscription/process_transcription.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/StartTranscriptionJob/common_lib.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/StartTranscriptionJob/common_lib.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/S3ProcessNewAudio/lambda_function.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/S3ProcessNewAudio/lambda_function.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: functions/source/StartTranscriptionJob/start_transcription.py to s3://voice-of-customer-gokazu-20201027/asset/functions/source/StartTranscriptionJob/start_transcription.py An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: templates/quickstart-connect-voci-base-pipeline.yaml to s3://voice-of-customer-gokazu-20201027/asset/templates/quickstart-connect-voci-base-pipeline.yaml An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: ./setup.ipynb to s3://voice-of-customer-gokazu-20201027/asset/setup.ipynb An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: templates/voci-lambda-portability.yaml to s3://voice-of-customer-gokazu-20201027/asset/templates/voci-lambda-portability.yaml An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n",
      "upload failed: templates/voci-cognito.yaml to s3://voice-of-customer-gokazu-20201027/asset/templates/voci-cognito.yaml An error occurred (AccessDenied) when calling the PutObject operation: Access Denied\n"
     ]
    }
   ],
   "source": [
    "!aws s3 sync . s3://{bucket}/asset --acl public-read"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "conda_python3",
   "language": "python",
   "name": "conda_python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
